### PR TITLE
fix up the workflow

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>

**What this PR does / why we need it**:
This PR tries another attempt at getting the docker push to work.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Related to #67 

**Special notes for your reviewer**:
**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: .github/main.workflow [hunk 1]
```diff
@@ -40,10 +40,9 @@ action "tag images" {
 
 action "push images" {
```
Comment (ashish-amarnath) [inline .github/main.workflow]: > It's recommended to not use the WORKDIR instruction in your Dockerfile. 
https://developer.github.com/actions/creating-github-actions/creating-a-docker-container/
We may want to change this https://github.com/kubernetes-sigs/cluster-api-provider-docker/blob/master/Dockerfile#L16

File: .github/main.workflow [hunk 2]
```diff
@@ -40,10 +40,9 @@ action "tag images" {
...
   uses = "actions/docker/cli@master"
-  runs = "sh -c"
   env = {
     IMAGE_NAME = "gcr.io/kubernetes1-226021/capd-manager"
   }
-  args = "source $HOME/.profile && docker push $IMAGE_NAME:latest && docker push $IMAGE_NAME:$IMAGE_REF && docker push $IMAGE_NAME:$IMAGE_SHA && docker push $IMAGE_NAME:$IMAGE_VERSION"
+  args = ["source $HOME/.profile && docker push $IMAGE_NAME:latest && docker push $IMAGE_NAME:$IMAGE_REF && docker push $IMAGE_NAME:$IMAGE_SHA && docker push $IMAGE_NAME:$IMAGE_VERSION"]
```
Comment (ashish-amarnath) [inline .github/main.workflow]: > If you provide args in a string, the string is split around whitespace.

https://developer.github.com/actions/managing-workflows/workflow-configuration-options/#actions-attributes

Do you think we'll have a better chance doing something like
```
runs = ["sh", "-c", "source $HOME/.profile && docker push $IMAGE_NAME:latest && docker push $IMAGE_NAME:$IMAGE_REF && docker push $IMAGE_NAME:$IMAGE_SHA && docker push $IMAGE_NAME:$IMAGE_VERSION"]
```

**Comment by ashish-amarnath** [comment]
Looks like a lot of trial and error. I wish I could run this to help you out here.

I am good trying out the changes you have here.
Approving w/ a hold for you to look at some suggestions I made based on some doc I read.
Feel free to cancel hold to try your change.
/hold
/lgtm

File: .github/main.workflow (hunk 2 above)
Comment (chuckha) [reply .github/main.workflow]: i think i shouldn't have removed the runs 🤔 oh well. i'll try that out next

Comment (chuckha) [reply .github/main.workflow]: one big problem is lack of actual logs. it just says "failed" at this point and doesn't tell me why

**Pull request merged**